packages/react-components/react-carousel-preview/library/etc/react-carousel-preview.api.md

```diff
@@ -167,8 +167,8 @@ export type CarouselProps = ComponentProps<CarouselSlots> & {
     defaultValue?: string;
     value?: string;
     onValueChange?: EventHandler<CarouselValueChangeData>;
-    circular?: Boolean;
-    peeking?: Boolean;
+    circular?: boolean;
+    peeking?: boolean;
```

Comment:
curious for my own learning - what's the difference between Boolean and boolean?

Reply:
I believe Boolean is an object that wraps boolean and has a get function when called as a variable, while boolean is a true primitive that can only output true | false

Reply:
nice!